From 988ceb0e56f57dfb55eebfa816c58cc9c28f5f48 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Mon, 03 May 2021 17:07:08 +0300
Subject: [PATCH] [SW-0]: fwupd: remove unneeded fit_base settings

Before the fwupd's state structure was introduced a getter and setter
used to access the mapped address. Now it's redundant should be removed.

Change-Id: I86bb1c349281a3d9d6182e7b16fb3777ea2afc6d
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/preboot/module_fwupd.c b/preboot/module_fwupd.c
index 9e3f389..8c83b25 100644
--- a/preboot/module_fwupd.c
+++ b/preboot/module_fwupd.c
@@ -305,7 +305,6 @@
 	u32 load_type;
 	int ret;
 
-	preboot_shm_set_offset((uintptr_t)state.map_offset);
 	ret = fit_get_load_type(&load_type);
 	if (ret)
 		return 1;
